src/Home.js

```diff
@@ -42,25 +42,34 @@ const Home = ({
 	const [isCopiedContainerId, setCopiedContainerId] = useState(false);
 	const [isCopiedObjectId, setCopiedObjectId] = useState(false);
 
-	const handleFile = (file, index) => {
+	const handleAllFiles = (files) => {
+		for (let i = 0; i < files.length; i += 1) {
+			handleFile(files[i], i === (files.length - 1));
+		}
+	};
+
+	const handleFile = (file, isClear = true, index) => {
 		if (index !== undefined) {
 			setFiles((filesTemp) => filesTemp.filter((item, indexItem) => indexItem !== index));
 		} else {
-			document.querySelector('input').value = '';
+			if (isClear) {
+				document.querySelector('input').value = '';
+			}
+
 			if (file === undefined) {
 				return;
 			}
 
-			if (file.size > 50 * 1024 * 1024) {
-				onModal('failed', 'Selected file is over 50Mb. We don\'t support such big files');
+			if (file.size > 200 * 1024 * 1024) {
```

Comment:
My suggestion is to precalculate this value and put it in a separate constant. And after using this constant here, instead of calculating this every time.